src/net.cpp

```diff
@@ -968,7 +968,7 @@ bool CConnman::AttemptToEvictConnection()
 
         LOCK(cs_vNodes);
         for (const CNode* node : vNodes) {
-            if (node->HasPermission(NetPermissionFlags::PF_NOBAN))
+            if (node->HasPermission(NetPermissionFlags::NoBan))
```

Comment:
meta: Could the rename be a scripted-diff?

Reply:
done